app/views/site/_view_gene_expression_heatmap.html.erb

```diff
@@ -60,6 +60,9 @@
     $('#dot-plot').data('rendered', false);
     function drawHeatmap(height) {
         $(window).off('resizeEnd');
+
+        $(window).on('resizeEnd', () => { $(window).trigger('resize.morpheus')})
```

Comment:
Does the `resize.morpheus` event have anything special associated with it?  It doesn't appear that when testing locally there is a functional difference between `resize` and `resize.morpheus`.

Reply:
Yup, `resize.morpheus` triggers only resizing the heatmap.  Implementation details are [here](https://github.com/cmap/morpheus.js/blob/a4055b4bb1c1e7907295a067275d6dca4b8137cd/src/ui/heat_map.js#L516).

I suppose my naive concern about `resizeEnd`-trigger-`resize` causing an infinitely loop is unfounded given your test.  That said, I lean towards `resize.morpheus` because that's consistent with how other widgets like scatter plots and violin plots handle `resizeEnd`.  They do so via widget-specific resize functions.